src/Components/test/E2ETest/Tests/FormsTest.cs

```diff
@@ -848,6 +848,47 @@ public void InputRangeAttributeOrderDoesNotAffectValue()
             Browser.Equal("210", () => rangeWithValueLast.GetDomProperty("value"));
         }
 
+        [Fact]
```

Comment:
I've only added tests verifying that `InputSelect` and `InputRadioGroup` work without a parent `EditForm`, since they contain most of the unique logic of the input components (other logic comes from `InputBase`). `InputRadioGroup` is a bit special in that it still calculates its own styling. The `InputSelect` test verifies that complicated binding scenarios still work (binding to arrays, boolean values).